src/jwplayer/players/CanonicalVideoPlayer/CanonicalVideoPlayer.tsx

```diff
@@ -35,17 +38,34 @@ const CanonicalVideoWrapper = styled.div<CanonicalVideoWrapperProps>`
 	}
 `;
 
-const CanonicalVideoPlayer: React.FC<CanonicalVideoPlayerProps> = ({ currentVideo, onComplete }) => {
+const CanonicalVideoPlayer: React.FC<CanonicalVideoPlayerProps> = ({ currentVideo, videoDetails, onComplete }) => {
 	const ref = useRef<HTMLDivElement>(null);
+	const adComplete = useAdComplete();
 	const onScreen = useOnScreen(ref, '0px', 1);
 	const isScrollPlayer = !onScreen;
 
+	useEffect(() => {
+		const payload = {
+			siteType: 'web',
+			isProduction: isOnBrowser() && !isLocalDevelopment(),
```

Comment:
Is "Dev" considered Prod in this case? It looks like in this case if its client-side, then this will resolve true for both a "prod" deployment and a "dev" deployments (whether its build-dev, dev or prod in the FAT ecosystem). Just wanted to make sure that this was the correct assumption.

Reply:
Ah, I just saw the isLocalDevelopment() definition. It looks like its the ENV variable that's set when a minified bundle is created, vs the un-minified bundle when you do local development.